Create universidade-de-fortaleza-abnt.csl

dayaaraujo opened the pull request:
I edited a code in ABNT, updating to the NBR 10520/2023 - Citation standard. Previously, the author's last name was in UPPERCASE; now, only the Initial is used. Example: Araújo (2024) or (Araújo, 2024).

dayaaraujo committed:
Create universidade-de-fortaleza-ABNT.csl

dayaaraujo committed:
Update universidade-de-fortaleza-ABNT.csl

Inicialmente coloquei o código inicial ao qual alterei e atualizei.

dayaaraujo committed:
Rename universidade-de-fortaleza-ABNT.csl to universidade-de-fortaleza-abnt.csl

POBrien333 commented on universidade-de-fortaleza-abnt.csl:
```diff
@@ -0,0 +1,741 @@
+    <id>http://www.zotero.org/styles/universidade-de-fortaleza-abnt</id>
+    <link href="http://www.zotero.org/styles/universidade-de-fortaleza-abnt" rel="self"/>
+    <link href="http://www.zotero.org/styles/associacao-brasileira-de-normas-tecnicas" rel="template"/>
+    <link href="https://www.abntcatalogo.com.br/" rel="documentation"/>
```
Can you update the link to a URL where the actual style is documented?

POBrien333 committed:
Merge branch 'master' into pr/7301

POBrien333 commented on universidade-de-fortaleza-abnt.csl:
```diff
@@ -0,0 +1,741 @@
+          </if>
+          <else>
+            <!--Se for outro tipo de documento aparecera o numero e depois a descricao "ed."-->
+            <text variable="edition" suffix=" ed."/>
```
The suffix label here doesn't make sense and can be removed.

POBrien333 commented on universidade-de-fortaleza-abnt.csl:
```diff
@@ -0,0 +1,741 @@
+        <text variable="publisher-place" suffix=". "/>
+      </else-if>
+      <else>
+        <text value="[s. l.]" font-style="italic" suffix=", "/>
```
replace with correct term.

POBrien333 commented on universidade-de-fortaleza-abnt.csl:
```diff
@@ -0,0 +1,741 @@
+        <else-if type="map">
+          <group>
+            <!--autor-->
+            <text macro="author" suffix=". "/>
```
instead of having all these suffixes, use groups with delimiters. This makes the style more robust.

POBrien333 reviewed (changes requested):
There are a variety of issues throughout the style. Please have a look.

dayaaraujo committed:
Update universidade-de-fortaleza-abnt.csl

Hello! I'm not very familiar with pull requests and XML, but I did my best to make the standard available in the most commonly used reference tools (Zotero, Mendeley, EndNote), considering that most other ABNT standards from other institutions are completely outdated. 

I hope it works now.
I'm a librarian at the University of Fortaleza, and it will be of great value. Thank you very much for your considerations and revisions.

Something very specific to ABNT, compared to APA, Chicago, Harvard, or other international standards, is the requirement to specify the place of publication for all documents. And when there isn't one, the author must indicate exactly "[s. l.]", in italics.

dayaaraujo committed:
Update universidade-de-fortaleza-abnt.csl

POBrien333 committed:
Merge branch 'master' into pr/7301

POBrien333 committed:
Merge branch 'patch-1' of https://github.com/dayaaraujo/styles into pr/7301

dayaaraujo committed:
Update universidade-de-fortaleza-abnt.csl

dayaaraujo committed:
Re-indent CSL styles
